nevergrad/optimization/optimizerlib.py

```diff
@@ -162,17 +169,22 @@ class ParametrizedOnePlusOne(base.ConfiguredOptimizer):
     # pylint: disable=unused-argument
     def __init__(
         self,
+        budget: Optional[int] = None,        
```

Comment:
this should not be here, it should be in the actual optimizer above

Reply:
thx!